subiquitycore/ui/views/network.py

```diff
@@ -220,9 +234,17 @@ def _build_model_inputs(self):
 
     def refresh_model_inputs(self):
         self.device_table.set_contents(self._build_model_inputs())
-        if isinstance(self._w, StretchyOverlay) and \
-           hasattr(self._w.stretchy, 'refresh_model_inputs'):
+        if isinstance(self._w, StretchyOverlay) and hasattr(
+                self._w.stretchy, 'refresh_model_inputs'):
             self._w.stretchy.refresh_model_inputs()
+        # we have heading, and then three lines per interface
+        # selectable line, extra line, whitespace line
+        # and focus ends up on the last whitespace line
+        # despite it, not being selectable. *derp*
+        current_focus = self.device_table.focus_position
```

Comment:
argh we need to do something better for this (and on filesystem screen too)

probably not on this branch though!

Reply:
https://trello.com/c/qrMBy4Ve